contracts/linear/src/validator_pool.rs

```diff
@@ -230,27 +255,70 @@ fn min3(x: u128, y: u128, z: u128) -> u128 {
 
 #[near_bindgen]
 impl LiquidStakingContract {
-    pub fn add_validator(&mut self, validator_id: AccountId, weight: u16) -> Validator {
+    pub fn set_whitelist_contract_id(&mut self, account_id: AccountId) {
```

Comment:
Should we move this method to `owner.rs`?

Reply:
sure